### PR TITLE
PUB-2191-Text update to identified instances of (s) and full stop in …

### PR DESCRIPTION
**Pull request by MindGames2015**
…CaTH

**Before creating a pull request make sure that:**

-   [ ] commit messages are meaningful and follow good commit message guidelines
-   [ ] README and other documentation has been updated / added (if needed)
-   [ X] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link

https://tools.hmcts.net/jira/browse/PUB-2191

### Change description

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/resources/locales/en/subscription-configure-list-confirmed.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "title": "Court Subscription(s) refined",
-    "header": "Court Subscription(s) refined",
-    "subheader": "Your subscription(s) has been amended successfully",
+    "header": "List types updated",
+    "subheader": " ",
```
Comment (KianKwa) [inline src/main/resources/locales/en/subscription-configure-list-confirmed.json:4]: subheader can be removed if it is not used. The njk will also need to be updated

File: src/main/resources/locales/cy/subscription-configure-list-confirmed.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "title": "Cadarnhau tanysgrifiad",
-    "header": "Tanysgrifiad wedi’i gadarnhau",
-    "subheader": "Rydych wedi’ch tanysgrifio yn llwyddiannus.",
+    "header": "Mathau y rhestrau wedi’u diweddaru",
+    "subheader": " ",
```
Comment (KianKwa) [inline src/main/resources/locales/cy/subscription-configure-list-confirmed.json:4]: SImilar to the English language file. Can remove subheader if unused anymore.

File: src/main/resources/locales/en/subscription-confirmed.json [hunk 3]
```diff
@@ -1,7 +1,6 @@
 {
     "title": "Subscription confirmation",
-    "header": "Subscription(s) confirmed",
-    "subheader": "Your subscription(s) has been added successfully",
+    "header": "Email subscriptions updated",
```
Comment (KianKwa) [inline src/main/resources/locales/en/subscription-confirmed.json:3]: Think subheader will also need to be removed from the njk file.

File: src/main/resources/locales/cy/party-name-search-results.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
     "title": "Canlyniad y chwilio",
-    "resultsCount": " Wedi dod i hyd i ganlyniad(au)",
+    "resultsCount": "Wedi canfod o ganlyniadau",
```
Comment (KianKwa) [inline src/main/resources/locales/cy/party-name-search-results.json:3]: Not sure if this is correct. The Welsh text in the ticket is as below, so you number of results needs to be injected between 'canfod' and 'o'. You can add a placeholer text then in the njk file replace the placeholder with the actual count. 
Wedi canfod [X] o ganlyniadau

File: src/main/resources/locales/en/case-reference-number-search-results.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
     "title": "Search result",
-    "subheader": "1 result successfully found",
+    "subheader": "1 found",
```
Comment (KianKwa) [inline src/main/resources/locales/en/case-reference-number-search-results.json:3]: I can't see the Welsh translation for this change. Guess we need to change Welsh too?

File: src/main/resources/locales/en/case-name-search-results.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
     "title": "Search result",
-    "resultsCount": " result(s) successfully found",
+    "resultsCount": " found",
```
Comment (KianKwa) [inline src/main/resources/locales/en/case-name-search-results.json:3]: Similar to the comment before...need Welsh changes too?

File: src/main/resources/locales/cy/party-name-search-results.json (hunk 4 above)
Comment (MindGames2015) [reply src/main/resources/locales/cy/party-name-search-results.json]: Committed changes

File: src/main/resources/locales/en/case-reference-number-search-results.json (hunk 5 above)
Comment (MindGames2015) [reply src/main/resources/locales/en/case-reference-number-search-results.json:3]: Committed changes

**Review by KianKwa — APPROVED** [review]

**Pull request merged**